src/covid_model_seiir_pipeline/lib/io/marshall.py

```diff
@@ -10,6 +10,8 @@
     MetadataKey,
 )
 
+POTENTIAL_INDEX_COLUMNS = ['location_id', 'date', 'age_start']
```

Comment:
Normally I would tell you constants should be tuples, not lists

But pandas is sometimes sensitive to types and tuples won't work, so instead I ask - can this be a tuple?

Reply:
It can be! I think I was using it directly at first, but I'm just iterating over it in current usage.